lib/labels/ec2/ec2.go

```diff
@@ -110,7 +110,7 @@ func (l *EC2) Sync(ctx context.Context) error {
 	tags, err := l.c.Client.GetTagKeys(ctx)
 	if err != nil {
 		if trace.IsNotFound(err) {
-			l.c.Log.Warningf("could not fetch tags, please ensure 'allow instance tags in metadata' is enabled on the instance")
+			l.c.Log.Debug("could not fetch tags, please ensure 'allow instance tags in metadata' is enabled on the instance")
```

Comment:
This is just spamming at a different log level, right? Should we change the behavior a bit? Some ideas:

- try up to a maximum number of times and then give up and assume the user doesn't want to fetch tags (at which point we stop logging it)
- only log this warning the first time we encounter it
- implement some sort of backoff so that retries get further apart (and therefore we see the log item less frequently)

WDYT?

Reply:
I like the idea of trying a max number of times and then assuming the user doesn't want tags.

Reply:
The only downside there is if they eventually enable the API on the AWS side, they have to restart Teleport to pick up the change.

Reply:
True, and I think that's less intuitive than EC2 labels working again once the API is enabled. I might try option 2 instead.

Reply:
@atburke I also think that option two makes the most sense. Can you just revert the log level to warning again? I don't think that anyone will notice one extra debug log now.